internal/cephfs/core/filesystem.go

```diff
@@ -29,11 +29,11 @@ import (
 // that interacts with CephFS filesystem API's.
 type FileSystem interface {
 	// GetFscID returns the ID of the filesystem with the given name.
-	GetFscID(context.Context, string) (int64, error)
+	GetFscID(ctx context.Context, fsName string) (int64, error)
```

Comment:
Can you please point to the linter that suggested this change? 
I am interested in reading about the suggestion.

Reply:
```go
internal/cephfs/core/filesystem.go:32:11: interface method GetFscID must have named param for type context.Context (inamedparam)
	GetFscID(context.Context, string) (int64, error)
	         ^
```